runtime/compiler/optimizer/EscapeAnalysis.cpp

```diff
@@ -6788,6 +6748,18 @@ void TR_EscapeAnalysis::makeContiguousLocalAllocation(Candidate *candidate)
                                                                           fieldEntry._typeSignature
                                                                           );
 
+         int32_t fieldSize = TR::Symbol::convertTypeToSize(fieldEntry._datatype);
+
+         if (fieldEntry._datatype.getDataType() == TR::Address && comp()->useCompressedPointers())
+            {
+            fieldSize = TR::Compiler->om.sizeofReferenceField();
+            }
```

Comment:
Good point.  This appears many times in existing code in Escape Analysis and in many places throughout the JIT compiler that are unrelated to Escape Analysis.  Would you mind if I opened a follow on clean-up issue to handle that, rather than dealing with it in this pull request?

Reply:
Opening a clean-up issue sounds good. Thank you!